pkg/loki/modules.go

```diff
@@ -1140,6 +1140,7 @@ func (t *Loki) initCompactor() (services.Service, error) {
 		level.Info(util_log.Logger).Log("msg", "-boltdb.shipper.compactor.shared-store not specified, initializing compactor to operator on the following object stores", "stores", strings.Join(stores, ", "))
 	}
 
+	t.Cfg.CompactorConfig.TableOperationsParallelism = t.Cfg.StorageConfig.MaxParallelismTableOps
```

Comment:
Mutating the config :/

Reply:
`CompactorConfig.TableOperationsParallelism` is just reusing `MaxParallelismTableOps` and this is the only place where an assignment for it happens. This way I don't have to change the whole compactor code to hold the Store config everywhere.